src/main/java/io/anserini/util/ExtractDocumentLengths.java

```diff
@@ -78,7 +78,7 @@ public static void main(String[] args) throws Exception {
 
     out.println("internal_docid\texternal_docid\tdoc_length\tunique_term_count\tlossy_doc_length\tlossy_unique_term_count");
     for (int i = 0; i < numDocs; i++) {
-      Terms terms = reader.getTermVector(i, Constants.CONTENTS);
+      Terms terms = reader.termVectors().get(i, Constants.CONTENTS);
```

Comment:
Ideally you'd fetch the `reader.termVectors()` object only once, before the loop, and then reuse it in the loop. (And similarly to other places that rely on stored fields and term vectors.)

Reply:
yeah, good point!

Reply:
Will have another PR fix that in different places (and perhaps  storedFields() too)